service-worker/prefetch-video/service-worker.js

```diff
@@ -36,9 +36,11 @@ self.addEventListener('install', function(event) {
     '/service-worker/prefetch-video/',
     '/service-worker/prefetch-video/index.js',
     '../../styles/main.css',
-    'static/video.mp4',
-    'static/video.webm',
-    'static/poster.jpg'];
+    'static/poster.jpg',
+    // The videos are stored remotely with CORS enabled.
+    'https://prefetch-video-sample.storage.googleapis.com/gbike.webm',
+    'https://prefetch-video-sample.storage.googleapis.com/gbike.mp4'
```

Comment:
May we remove `static/video.mp4` and `static/video.webm` files from the repo as well?